ts/twitter/src/app/login/_components/login-modal.stories.tsx

```diff
@@ -0,0 +1,163 @@
+import { err, ok } from "@/lib/actions/types";
+import { HTTP_STATUS } from "@/lib/constants/http-status";
+import type { Meta, StoryObj } from "@storybook/react";
+import { expect, userEvent, within } from "@storybook/test";
+import { login } from "#src/lib/actions/mocks/login.mock";
+import { LoginModal } from "./login-modal";
+
+const meta: Meta<typeof LoginModal> = {
+  title: "Features/LoginModal",
+  component: LoginModal,
+  tags: ["!autodocs"],
+};
+
+export default meta;
+type Story = StoryObj<typeof LoginModal>;
+
+export const AccountForm: Story = {
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+
+    await expect(canvas.findByText("Login to X")).resolves.toBeInTheDocument();
```

Comment:
```suggestion
    expect(canvas.getByText("Login to X")).toBeInTheDocument();
```

`findBy` is used to wait for an element to appear but the change to the DOM might not happen immediately.
Please update the other relevant parts accordingly as well.